code/game/objects/explosion.dm

```diff
@@ -112,10 +114,11 @@ proc/explosion(turf/epicenter, devastation_range, heavy_impact_range, light_impa
 			var/throw_dir = get_dir(epicenter,T)
 			for(var/obj/item/I in T)
 				spawn(0) //Simultaneously not one at a time

```

Comment:
TBH this spawn() probably causes a bunch of lag too, and I don't think it actually makes that much difference.